src/content/add-to-app/ios/project-setup.md

```diff
@@ -273,89 +272,95 @@ some/path/MyApp/
-application in Xcode.  There are multiple ways to do
-this&mdash;use the method that is best for your project.
+#### Procedures
+
+You can either link or embed the generated frameworks
+into your existing app in Xcode.
+Use the method that is best for your project.
 
-#### Link on the frameworks
+##### Link on the frameworks
 
 For example, you can drag the frameworks from
-`some/path/MyApp/Flutter/Release/` in Finder
-into your target's **Build
-Settings > Build Phases > Link Binary With Libraries**.
+`/path/to/MyApp/Flutter/Release/` in the Finder into your target's
+**Build Settings > Build Phases > Link Binary With Libraries**.
 
 In the target's build settings, add `$(PROJECT_DIR)/Flutter/Release/`
 to the **Framework Search Paths** (`FRAMEWORK_SEARCH_PATHS`).
 
-{% render docs/app-figure.md, image:"development/add-to-app/ios/project-setup/framework-search-paths.png", alt:"Update Framework Search Paths in Xcode" %}
+{% render docs/captioned-image.liquid, image:"development/add-to-app/ios/project-setup/framework-search-paths.png", caption:"Update Framework Search Paths in Xcode" %}
 
 :::tip
-To use the simulator, you will need to 
-embed the Debug version of the Flutter frameworks in your
-Debug build configuration. To do this 
-you can use `$(PROJECT_DIR)/Flutter/$(CONFIGURATION)`
-in the **Framework Search Paths** (`FRAMEWORK_SEARCH_PATHS`)
-build setting. This embeds the Release frameworks in the Release configuration, 
+To use the simulator, embed the Debug version of the Flutter frameworks
+in your Debug build configuration.
+To do this, use `$(PROJECT_DIR)/Flutter/$(CONFIGURATION)`
+in the **Framework Search Paths** (`FRAMEWORK_SEARCH_PATHS`) build setting.
+This embeds the Release frameworks in the Release configuration,
 and the Debug frameworks in the Debug Configuration.
 
-You must also open `MyApp.xcodeproj/project.pbxproj` (from Finder) 
+You must also open `MyApp.xcodeproj/project.pbxproj` (from Finder)
 and replace `path = Flutter/Release/example.xcframework;`
 with `path = "Flutter/$(CONFIGURATION)/example.xcframework";`
-for all added frameworks. (Note the added `"`.)
+for all added frameworks. The double quotes (`"`) are necessary.
 :::
 
-#### Embed the frameworks
+##### Embed the frameworks
 
-The generated dynamic frameworks must be embedded
-into your app to be loaded at runtime.
+To load the generated dynamic frameworks at runtime,
+embed them into your app.
 
 :::important
 Plugins might produce [static or dynamic frameworks][].
 Static frameworks should be linked on, but never embedded.
-If you embed a static framework into your application,
-your application is not publishable to the App Store
-and fails with a
+If you embed a static framework into your iOS app,
+you can't publish that app to the App Store.
+Publishing fails with a
 **Found an unexpected Mach-O header code** archive error.
 :::
 
-After linking the frameworks, you should see them in the 
+After linking the frameworks, they should display in the
```

Comment:
Consider "appear" to make it clearer that this won't happen until after you add linking:

```suggestion
After linking the frameworks, they should appear in the
```